js/src/forum/components/PostQuoteButton.js

```diff
@@ -1,27 +1,29 @@
-import Button from 'flarum/components/Button';
-import extract from 'flarum/utils/extract';
+import Fragment from 'flarum/Fragment';
+import icon from 'flarum/helpers/icon';
 
 import reply from '../utils/reply';
 
-export default class PostQuoteButton extends Button {
+export default class PostQuoteButton extends Fragment {
```

Comment:
`PostQuoteButton` should be moved to the `fragments` folder like `AutocompleteDropdown`

Reply:
Fixed!